students/JeroenSweerts/Lesson9/mailroom5.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+
+class Donor():
+    def __init__(self, name=None):
+        self._name = name
+        self._donations = []
+
+    @property
+    def name(self):
+        """Property for the donor's name."""
+        return self._name
+
+    @name.setter
+    def name(self, val):
+        """Setter for the donor's name."""
+        self._name = val
+
+    @property
+    def donations(self):
+        """Property for the donor's list of donations."""
+        return self._donations
+
+    @donations.setter
+    def donations(self, val):
+        """Setter for the donor's list of donations."""
+        self._donations = val
+
+
+class Donor_Collection(dict):
+    def __init__(self, donors):
+        self._donors = donors
```

Comment:
This overwrites the built-in `dict` class's constructor without calling `super().__init__()`